main.go

```diff
@@ -53,23 +56,21 @@ func startAMQChecks() {
 }
 
 func startSSOChecks() {
-	url := "https://keycloak-edge-redhat-rhmi-rhsso.apps.sso-128.y3z4.s1.devshift.org"
-	user := "admin"
-	pwd := "NH0trX4G93snKw=="
+	url := os.Getenv(envVarURL)
+	user := os.Getenv(envVarUser)
+	pwd := os.Getenv(envVarPassword)
 	realm := "master"
 	if url != "" && user != "" && pwd != "" && realm != "" {
 		log.WithFields(log.Fields{
 			"serverURL": url,
-			"user":      user,
-			"password":  pwd,
 			"realmName": realm,
-		}).Info("Start SSO Checks!!")
+		}).Info("Start SSO Checks")
 		c := &SSOChecks{
 			serverURL: url,
 			user:      user,
 			password:  pwd,
 			realmName: realm,
-			interval:  1 * time.Second,
+			interval:  7 * time.Second,
```

Comment:
Interval is set to 7s for avoiding the risk of blocking the login action by the rate limit of SSO.